chain/network/src/store/mod.rs

```diff
@@ -0,0 +1,185 @@
+#![allow(dead_code)] //TODO: remove, it will be in use in the next PR.
+/// Store module defines atomic DB operations on top of schema module.
+/// All transactions should be implemented within this module,
+/// in particular schema::StoreUpdate is not exported.
+use near_network_primitives::types::{Edge, KnownPeerState};
+use near_primitives::network::{AnnounceAccount, PeerId};
+use near_primitives::types::AccountId;
+use std::collections::HashSet;
+use tracing::debug;
+
+mod schema;
+
+pub use schema::Error;
+
+/// Store allows for performing synchronous atomic operations on the DB.
+/// In particular it doesn't implement Clone and requires &mut self for
+/// methods writing to the DB.
+pub struct Store(schema::Store);
```

Comment:
So this is effectively `pub(crate)`, because the parent module is `pub(crate)`, right? Effective pub vs effective pub(crate) is a big deal, and it's a bit hard to tell which case are we in here, as this requires reading the mod declaration in the other file. 

That's why I personally favor repating `pub(crate)` declarations. Telling, locally, whether the thing is really visible outside of CU is very helpful. 

This is not in the style guide yet, because we are somewhat inconsistent about this. But I guess I should go ahead and propose this. 

(but, until there's a documented consensus, no strong opinion :) )

Reply:
Damn, that's what you get when you mix 2 different concepts:
- module visibility (which modules see the given module)
- module API (which types/functions are visible outside of the module)
It is not like that you can just say pub(crate) and the thing will be automatically visible in the whole crate - the parent crate will have to reexport it anyway (in this particular case the parent is the whole crate, so it doesn't apply though).

IMO the crate shouldn't be even allowed to restrict visiblity in a different way than pub/private. The whole module is just an implementation detail of the parent, and it is parent's responsibility to define its public API.

Reply:
anyway, done